### ISSUE TITLE
Terminus plugin

### ISSUE THREAD

**Issue by EmilienLeroy** [tsedio/tsed#1259]
## Informations
**Is your feature request related to a problem? Please describe.**
Adding an implementation of [terminus](https://github.com/godaddy/terminus) library. This can be used to create a route which determine if our app is health or unhealth. 

**Describe the solution you'd like**
Add a ``@Health`` decorator based on the terminus library. Nest have his own implementation of the terminus library for [example](https://docs.nestjs.com/recipes/terminus).

```ts
import { Health } from "@tsed/terminus";

@Controller("/health")
export class HealthController {
    @Health()
    public health() {
        // determine is the app is health or not
    }
}
```

**Describe alternatives you've considered**
Currently, I have just create a simple controller and service which check if my database is up or not. This work but it does not cover all the cases covered by terminus  :

### HealthService.ts
```ts
import { Inject, Service } from '@tsed/di';
import { MongooseService } from '@tsed/mongoose';
import { version } from '../../package.json';

@Service()
export default class HealthService {
  @Inject(MongooseService)
  public mongoose: MongooseService;

  public health() {
    const { readyState } = this.mongoose.get();

    if (readyState === 0 || readyState === 2 || readyState === 3) {
      return Promise.reject(new Error());
    }

    return Promise.resolve({ version });
  }
}
```
### HealthController.ts
```ts
import { Controller, Get, Inject } from '@tsed/common';
import HealthService from './HealthService';

@Controller('/health')
export default class HealthController {
  @Inject(HealthService)
  private healthService: HealthService;

  @Get()
  public async health() {
    try {
      const { version } = await this.healthService.health();
      return {
        status: 'ok',
        version,
      };
    } catch (error) {
      return { status: 'error' };
    }
  }
}
```

**Comment by Romakita** [comment]
Hello @EmilienLeroy,

How I can help you? I think, you want to create a new Ts.ED package on this repository, right? 
I can help you to do that, if you are motivated to create the PR around this feature. 

Tell me :)
See you
Romain

**Comment by EmilienLeroy** [comment]
Hello @Romakita 

I think a new package can be a good solution for implement this library. I can try to do it myself with some help. Do you have a discord server or something else ?

**Comment by Romakita** [comment]
Here is the slack team ;) https://join.slack.com/t/tsed-framework/shared_invite/zt-ljtmbq2u-ln3FoEd4m4Oe8rIJ1~WjdQ

**Comment by EmilienLeroy** [comment]
Thanks !
I don't have lot of free time now but when I can, I will take a look to this feature !

**Comment by yuki-takei** [comment]
Our implementation for graceful-shutdown with terminus.
It's not well-tested code, but I hope it helps you.

https://github.com/weseek/growi/commit/7246df4f761d5bd50f60f23fe1d48faf339e70b5

**Comment by EmilienLeroy** [comment]
Hi @yuki-takei 
Thanks for this, it will help me to implement completely the terminus plugin !

**Comment by Romakita** [comment]
Released with v6.63.0
